example/hello_world.py

```diff
@@ -81,10 +81,21 @@ def dimensions(test):
 
 def attachments(test):
   test.Attach('test_attachment', 'This is test attachment data.')
-  with tempfile.NamedTemporaryFile() as f:
-    f.write('This is a file attachment')
-    f.flush()
-    test.AttachFromFile(f.name)
+  # Windows doesn't allow reading from an open NamedTemporaryFile, so we
+  # have to do this the hard way.  Typically in a real test, you would have
+  # invoked some external program that generated an output file.
```

Comment:
As discussed in meeting, let's just make a static file in example/ for this instead of generating on the fly, so as to keep this Windows nightmare code out of the example. We don't want users to follow our example and write stuff like this.